kotest-assertions/src/commonMain/kotlin/io/kotest/properties/gens.kt

```diff
@@ -619,6 +622,11 @@ fun <K, V> Gen.Companion.map(genK: Gen<K>, genV: Gen<V>, maxSize: Int = 100): Ge
    }
 }
 
+/**
+ * @return a stream of values, where each value is a [Map] of [Pair]s from the given [gen]
+ *   the size of the [Map] is bounded between [0, [maxSize])
+ */
+@JvmOverloads
```

Comment:
Why was this JvmOverloads added?

Reply:
My thought was to match the definition on `map` where the `key` and `value` are separate `Gen`s 

https://github.com/kotlintest/kotlintest/blob/a3a9065bc2d69d6184f8e6e1ea22924c2c07af3b/kotest-assertions/src/commonMain/kotlin/io/kotest/properties/gens.kt#L604-L610

And/or if the library wants to support Java having the condensed signature